docs/static/ds4/docs.css

```diff
@@ -1,2 +1,2 @@
}.highlight .gd{background-color:#fcc;border:1px solid #c00}.highlight .ge{font-style:italic}.highlight .gr{color:#f00}.highlight .gh{color:#030}.highlight .gi{background-color:#cfc;border:1px solid #0c0}.highlight .go{color:#aaa}.highlight .gp{color:#009}.highlight .gu{color:#030}.highlight .gt{color:#9c6}.highlight .kc{color:#069}.highlight .kd{color:#069}.highlight .kn{color:#069}.highlight .kp{color:#069}.highlight .kr{color:#069}.highlight .kt{color:#078}.highlight .m{color:#f60}.highlight .s{color:#d44950}.highlight .na{color:#4f9fcf}.highlight .nb{color:#366}.highlight .nc{color:#0a8}.highlight .no{color:#360}.highlight .nd{color:#99f}.highlight .ni{color:#999}.highlight .ne{color:#c00}.highlight .nf{color:#c0f}.highlight .nl{color:#99f}.highlight .nn{color:#0cf}.highlight .nt{color:#2f6f9f}.highlight .nv{color:#033}.highlight .ow{color:#000}.highlight .w{color:#bbb}.highlight .mf{color:#f60}.highlight .mh{color:#f60}.highlight .mi{color:#f60}.highlight .mo{color:#f60}.highlight .sb{color:#c30}.highlight .sc{color:#c30}.highlight .sd{color:#c30;font-style:italic}.highlight .s2{color:#c30}.highlight .se{color:#c30}.highlight .sh{color:#c30}.highlight .si{color:#a00}.highlight .sx{color:#c30}.highlight .sr{color:#3aa}.highlight .s1{color:#c30}.highlight .ss{color:#fc3}.highlight .bp{color:#366}.highlight .vc{color:#033}.highlight .vg{color:#033}.highlight .vi{color:#033}.highlight .il{color:#f60}.css .o,.css .o + .nt,.css .nt + .nt{color:#999}@media (min-width:600px){#header-small{display:none}#header-large{box-sizing:border-box;display:block;border-right:1px solid #ececec;height:100%;overflow-y:scroll;padding:0 16px 64px 16px;width:200px}#header-large h1{margin-top:34px;margin-bottom:38px}#header-large h1 a{display:block}#header-large h1 span[role="img"]{display:inline-block}#header-large h2{font-weight:400;font-size:1.062rem;margin-bottom:8px;margin-top:24px}#header-large ul{list-style-type:none;margin:0 16px;padding:0;width:100%}#header-large li{margin:6px 0}#header-large a{font-weight:400;font-size:0.9375rem;color:#555;text-decoration:none;white-space:nowrap;width:100%}#header-large a:hover{color:#333;text-decoration:underline}#main{margin-right:16px;margin-left:210px;padding-top:0}#main > div:first-child h2{font-weight:100;font-size:3.5rem;margin-bottom:8px;margin-top:16px;padding:0}#main > div:first-child h3{font-weight:400;font-size:1.062rem}#main > div:not(:first-child) > h2{font-weight:100;font-size:3.125rem;padding-top:20px;margin-top:44px}#main > div:not(:first-child) > h3{font-weight:400;font-size:1.5rem}#main > div:not(:first-child) > .grid h3{font-weight:400;font-size:1.062rem}#main > div:not(:first-child) > h4{font-weight:400;font-size:1.25rem}#main > div:not(:first-child) > h5{font-weight:400;font-size:0.875rem}footer[role="contentinfo"]{text-align:left;width:200px}figure.highlight pre{padding:16px 16px 0}.demo__inner{margin:16px}.demo--grid{padding-bottom:16px}.cols{column-count:3;-moz-column-count:3;-webkit-column-count:3}}
```

Comment:
Not sure why this is changed. Maybe we need to lock down on our version of autoprefixer, or whatever is changing this.

Reply:
Is it a build environment issue (mine v. yours v. someone else's)?

Reply:
Yep. We basically all have different versions of devDeps installed, it appears.

Reply:
For example, `-webkit-box-sizing:border-box` prefix has been removed.